test/lib/fixtures/resolutions/yarn1/expected-tree.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "adm-zip": {
+      "labels": {
+        "scope": "prod"
+      },
+      "name": "adm-zip",
+      "version": "0.4.7"
```

Comment:
This is actually not correct. The actual version of `adm-zip` that has been installed is `0.4.8`, due to `resolutions`. I'm not attempting to fix it in this PR.